dcrutil/README.md

```diff
@@ -0,0 +1,26 @@
+dcrutil
+=======
+
+
+[![Build Status](http://img.shields.io/travis/decred/dcrutil.svg)](https://travis-ci.org/decred/dcrd)
```

Comment:
This is still incorrect.   Needs to be `[http://img.shields.io/travis/decred/dcrd.svg](https://travis-ci.org/decred/dcrd)`.

Reply:
sorry, fixing